pyhdx/panel/configurations.py

```diff
@@ -0,0 +1,23 @@
+import os
+import configparser
+
+class ConfigurationSettings(object):
```

Comment:
Add a numpy-style class-level docstring describing `__init__` args and their types